README.md

```diff
@@ -74,6 +73,52 @@ subsCard.init();
 ```
 This will instantiate all subs-cards within the document. Alternatively you can pass in a HTMLElement, or String to be used as a selector to limit the scope of the instantiated subs cards to only child elements of the passed in scope.
 
+## Migration Guide
+
+### Upgrading from v1.x.x to v2.x.x
+
+o-subs-card v2.x.x changes the underlying style of all cards, removes two themes, adds a new B2B variation, and has updated markup. In this version, the expanding functionality is limited to the mobile and tablet viewports.
+
+The following themes have been removed:
+
+```diff
+- oSubsCardPremium / 'o-subs-card--premium'
+- oSubsCardPremiumPlus / 'o-subs-card--premium-plus'
+```
+
+And the available themes (and their respective classes) are now:  
+
+```diff
++ oSubsCardB2B / 'o-subs-card--b2b'
+oSubsCardDiscount / 'o-subs-card--discount' //no change
+```
+
+The markup has been rearranged, and  some classes have been removed.
```

Comment:
I think it might be helpful here to write the changes as a bulleted list to coincide with the diff.

- The charge value is now wrapped in a `div`.
- The charge value is now placed below the select button.
...etc

Just a suggestion though.